x-pack/plugins/ml/server/routes/data_frame_analytics.ts

```diff
@@ -609,12 +609,12 @@ export function dataFrameAnalyticsRoutes({ router, mlLicense, routeGuard }: Rout
   /**
    * @apiGroup DataFrameAnalytics
    *
-   * @api {post} /api/ml/data_frame/analytics/job_exists Check whether jobs exists in current or any space
+   * @api {post} /api/ml/data_frame/analytics/jobs_exist Check whether jobs exists in current or any space
    * @apiName JobExists
    * @apiDescription Checks if each of the jobs in the specified list of IDs exist.
```

Comment:
Not changed in the is PR, but should it be `exists` here with a trailing `s`?

Reply:
The `api` is correct here - it matches the endpoint `/api/ml/data_frame/analytics/jobs_exist`. Just the doc needs changing to be `Check whether jobs exist....`

Reply:
Edited the `apiDescription` in https://github.com/elastic/kibana/pull/126914/commits/61a27fc1aa19467aa978b9b9682271aa30fdafe8.